db/db_impl.cc

```diff
@@ -3064,6 +3066,14 @@ Status DBImpl::GetLatestSequenceForKey(SuperVersion* sv, const Slice& key,
     return Status::OK();
   }
 
+  SequenceNumber min_seq_in_imm =
```

Comment:
I wonder if we can apply the same optimization to skip checking the immutable memtables as well. Most of the snapshots are so recent that looking in the active memtable should be sufficient.